lib/include/scion/scion.h

```diff
@@ -42,149 +42,149 @@ enum scion_error {
 	/**
 	 * Something went wrong.
 	 */
-	SCION_GENERIC_ERR = -1,
+	SCION_ERR_GENERIC_ERR = -1,
```

Comment:
maybe rename this to `SCION_ERR_GENERIC` or `SCION_ERR_UNKNOWN`

Reply:
done